documentation/docs/25-build-and-deploy/55-single-page-apps.md

```diff
@@ -2,20 +2,21 @@
 title: Single-page apps
 ---
 
-You can turn any SvelteKit app, using any adapter, into a fully client-rendered single-page app (SPA) by disabling SSR at the root layout:
+You can turn a SvelteKit app into a fully client-rendered single-page app (SPA) by disabling SSR at the root layout and specifying a fallback page.
 
+> SPA mode is highly discouraged for public-facing websites (i.e. not behind a login or only used internally at a company). It has a large performance impact by forcing two network round trips before rendering can begin. This may be acceptable if you are serving a corporate application from the local network where the network round trip is very fast, but probably is not for most websites on the internet especially when considering the latency of mobile devices. It also harms SEO by increasing the likelihood that your site will be downranked for performance (SPAs are much more likely to fail core web vitals), excluding search engines that don't render JS, and causing your site to receive less frequent updates from those that do. And finally, it makes your app inaccessible to users if JavaScript fails or is disabled (which happens [more often than you probably think](https://kryogenix.org/code/browser/everyonehasjs.html)).
```

Comment:
Now that this section is so long I'm wondering if we should remove the `>`, i.e. not make it one big red box.

Reply:
I just tried both locally and think the `>` really is appropriate. It's an aside/warning and looks more consistent with how we treat that content elsewhere to leave it in the box. It's quite easy to gloss over it otherwise, so I think it'd be going backwards in terms of educating users to remove it from a box